frontend-react/src/components/DataDashboard/FacilityProviderSubmitterDetails/FacilityProviderSubmitterSummary.module.scss

```diff
@@ -0,0 +1,5 @@
+.FacilityProviderSubmitterSummary {
+    hr {
+        border: 1px solid color('base-lighter');
```

Comment:
are these color tokens working for you? i have a PR open that'll make the USWDS library work with css modules